elasticsearch-singledocument/src/main/java/org/vertexium/elasticsearch/score/ScoringStrategy.java

```diff
@@ -39,4 +40,12 @@ protected GraphConfiguration getGraphConfiguration() {
     public List<String> getFieldNames() {
         return new ArrayList<>();
     }
+
```

Comment:
No change needed here but, Is there any particular reason why only certain methods are required to be defined by ScoringStrategy(s) implementations?  Just a bit concern that someone may develop a ScoringStrategy not implementing those methods, and a user of that strategy downstream uses them (because they are available) expecting them to do something.